model/test_results.go

```diff
@@ -478,18 +486,18 @@ func (t *TestResults) GetPrestoBucket(ctx context.Context) (pail.Bucket, error)
 
 // TestResultsInfo describes information unique to a single task execution.
 type TestResultsInfo struct {
-	Project                string `bson:"project,omitempty"`
-	Version                string `bson:"version,omitempty"`
```

Comment:
What's the purpose of dropping all the "omitempty"?

Reply:
These are "required" fields, it is invalid for the to be empty.

Reply:
Thanks!